akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -348,7 +347,8 @@ private[stream] final class GraphInterpreter(
   val outHandlers: Array[OutHandler], // Lookup table for the outHandler of the connection
   val logics: Array[GraphStageLogic], // Array of stage logics
   val onAsyncInput: (GraphStageLogic, Any, (Any) ⇒ Unit) ⇒ Unit,
-  val fuzzingMode: Boolean) {
+  val fuzzingMode: Boolean,
+  val context: AnyRef) {
```

Comment:
Actually ActorGraphInterpreter can have several shells - initial and for all sub streams (```ActorGraphInterpreter.registerShell```). I believe that  ActorGraphInterpreter as Actor executes all its shells in one thread so we can accelerate messages from initial shell to substream shells and back. I can make context parameter as ActorRef to make it typed. Let me know if my assumptions are not correct.